libqtile/core/manager.py

```diff
@@ -153,10 +154,6 @@ def _configure(self):
         self._process_screens()
         self.current_screen = self.screens[0]
 
-        self.conn.flush()
-        self.conn.xsync()
-        self.core._xpoll()
```

Comment:
I wonder if this means that the manager shouldn't have the connection any more, since it's really an x11 thing?

Reply:
Yes for sure. There are _lots_ of things that need to be changed both in the manager and diffused throughout the code.

My approach for resolving all of these is following chronologically what happens when the WM starts up, and so the commits loosely follow that logic. Currently at setting up mouse button grabs and listeners (`Qtile._configure`). I think many things will simply require moving some of the logic from `manager.py` into the X backend and adding an equivalent method to the cores, but some things like `window.Window`'s hard-coded x window references are going to be a pain.